manual/users/users_gsa.tex

```diff
@@ -0,0 +1,260 @@
+
+
+\section{Application}
+\label{sec:app}
+
+We consider the following equation for a straight line:
+
+\be y = mx + c \ee
+
+The slope, $m$ and the y-intercept, $c$ are considered to be uniformly distributed in the intervals,
+[2, 5] and [3, 7] respectively. For reference purposes, we provide an algorithm followed by the
+C++ code which interfaces with QUESO to generate the required data for GSA as follows.
+In order to estimate the first order effect and the total effect sensitivity indices, we solve
+the forward problem in QUESO to generate the required set of data. Specifically, we need to generate
+(2$n$+2) data files for $n$ model parameters. Hence, in the present case, we need 6 data files as 
+listed and described below. Note that the pseudorandom samples pertaining to the  individual model parameters
+are given by their respective columns. In this case, column 1 corresponds to the slope, $m$ and column 2
+corresponds to the y-intercept, $c$. Moreover, $y$ in the above equation is regarded as the QoI. 
+
+\begin{enumerate}
+\item \texttt{qoi\_samplesA.txt}: Pseudo-random samples and corresponding QoI estimates, regarded as set $\bm A$. 
+\item \texttt{qoi\_samplesB.txt}: Pseudo-random samples and corresponding QoI estimates, regarded as set $\bm B$.
+\item \texttt{m\_qoi\_samplesAi.txt}: All columns from set $\bm A$ except the $i^{th}$ ($i$=1) column which is from
+set $\bm B$ and corresponding QoI estimates.
+\item \texttt{m\_qoi\_samplesBi.txt}: All columns from set $\bm B$ except the $i^{th}$ ($i$=1) column which is from
+set $\bm A$ and corresponding QoI estimates.
+\item \texttt{c\_qoi\_samplesAi.txt}: All columns from set $\bm A$ except the $i^{th}$ ($i$=2) column which is from
+set $\bm B$ and corresponding QoI estimates.
+\item \texttt{c\_qoi\_samplesBi.txt}: All columns from set $\bm B$ except the $i^{th}$ ($i$=2) column which is from
+set $\bm A$ and corresponding QoI estimates.
+\end{enumerate}
+
+In the above list, let use denote files in 3---6 as the set of derived files.  The following algorithm provides
+a sequence of steps as well as snippets of code which could be used to
+generate the set of data files to compute the sensitivity indices. Relevant source
+files have also been included later in this section.  
+
+\begin{algorithm}
+\caption*{\textbf{Algorithm}:~Generating data for GSA}
+\begin{algorithmic}[1]
+\Procedure{Solving SFP with QUESO}{}
+\BState Instantiate a QoI object (\texttt{qoi\_mc}): 
+\Statex \texttt{Qoi\_mc<> qoi\_mc("qoi\_", paramDomain, qoiSpace);}
+\BState Instantiate the forward problem (\texttt{fp\_mc}):
+\Statex \texttt{QUESO::StatisticalForwardProblem<> fp\_mc("", NULL, priorRv, qoi\_mc, qoiRv);} 
+\BState Solve the forward problem to generate the data file, \texttt{qoi\_samplesA.txt} i.e. set $\bm A$. 
+\Statex \texttt{fp\_mc.solveWithMonteCarlo(NULL);}
+\BState Repeat steps 2---4 to generate the data file, \texttt{qoi\_samplesB.txt} i.e. set $\bm B$. 
```

Comment:
[Use `--` for ranges](http://tex.stackexchange.com/questions/3819/dashes-vs-vs).